README.adoc

```diff
@@ -44,19 +47,6 @@ gem install rouge
 ----
 ====
 
-[TIP]
```

Comment:
I think you should keep this tip, as we have similar in other style guides.

Reply:
Okay, great!